Source/WebInspectorUI/UserInterface/Views/CookiePopover.js

```diff
@@ -33,10 +33,12 @@ WI.CookiePopover = class CookiePopover extends WI.Popover
         this._valueInputElement = null;
         this._domainInputElement = null;
         this._pathInputElement = null;
+        this._partitionKeyInputElement = null;
```

Comment:
where is this used?  ideally we'd at least show the `partitionKey` (i.e. a disabled input) in the cookie popover so that developers can see it even if they can't edit it

Reply:
That is stale and was leftover. I'll remove it.